audit/ask-cal/source/constants/categories.ts

```diff
@@ -0,0 +1,14 @@
+export type CodeExampleCategory =  
+  | "Syntax example"
+  | "Usage example"
+  | "Non-MongoDB command"
+  | "Example return object"
+  | "Example configuration object";
```

Comment:
A type works here, but it almost feels like an enum might be better? We're essentially doing the same thing as with the `const` on line 8, no? The ergonomic difference is that an enum here would let us type `CodeExampleCategory.` and intellisense would fill out all of our options.